src/templates/type/moment-select.html

```diff
@@ -7,7 +7,7 @@
                         class="select"
                         ng-model="hours"
                         ng-disabled="disable"
-                        ng-options="hour as hour for hour in hourChoices">
+                        ng-options="hour as (trailingHours + hour).slice(-(trailingHours.length)) for hour in hourChoices">
```

Comment:
This is confusing to read. Is there another way you can accomplish the same task?

I'm not sure what you need here so I can't offer any alternatives. Could you explain what this is doing please?